src/pages/blog/starlingx-single-core-platform.md

```diff
@@ -0,0 +1,60 @@
+---
+templateKey: blog-post
+title: StarlingX Single-Core Platform
+author: Davi Frossard
+date: 2023-05-25T11:14:00.000Z
+category: 
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+Introducing the Single-Core Platform on StarlingX 8.0, for All-In-One (AIO) deployments. <!-- more -->
+
+The [StarlingX 8.0](https://www.starlingx.io/blog/starlingx-release-8/) release came out recently with a new feature: The single physical core platform for AIO deployments. In this blog post, we will explore the changes made to the system to make this possible, and the benefits it offers.
+
+### What is StarlingX Single-Core Platform?
+
+Platforms that feature multiple cores and processors often consume significant amounts of computational resources. By enabling the single-core platform functionality, the system can allocate more resources to efficiently manage larger workloads. This optimization leads to enhanced resource availability for end-user applications. To prevent process starvation, the feature can only be enabled with two logical cores (Hyper-Threading enabled), ensuring proper overall parallelism and robustness for the system. To better illustrate this concept, see the diagram below:
+
+![StarlingX Single-Core Platform Diagram](/img/single-core-diagram.png)
+
+The use of a single physical core for platform function is only suitable for Intel® 4th Generation Xeon® Scalable Processors or above and should not be configured for previous Intel® Xeon® CPU families. For All-In-One systems with older generation processors, two physical cores (or more) must be configured.
+
+With the introduction of StarlingX 8.0, the default number of platform cores for AIO deployments has been set to 1. This adjustment requires a careful balance to ensure that the performance remains unaffected despite having only one platform core. In the following sections, we will dive into some specific changes made to each service. By examining these modifications, we will gain a comprehensive understanding of how the system can operate in a single-core environment while preserving performance and stability.
+
+### Platform Core Adjustments
+
+## System Inventory
+
+# Periodic and Runtime Tasks
+
+Some of the sysinv-conductor and sysinv-agent periodic tasks have been redesigned. The main changes focused on refactoring legacy code, increasing time intervals, and converting periodic tasks to on-demand tasks, whenever possible.
+
+# Remote Procedure Calls
+
+In previous versions, RPC communication between sysinv-api, sysinv-conductor and sysinv-agent processes are performed using an openstack RPC library, with RabbitMQ as the communication transport layer. To decouple sysinv from RabbitMQ, using a brokerless solution instead, a new backend was developed. This backend was implemented using a library built on top of ZeroMQ for message queuing and MessagePack for data serialization.
+
+## Scale-Down Services
```

Comment:
```suggestion
## Shrink Service Footprint
```

Reply:
I prefer to keep Scale-Down to match with other documents, such the starlingx spec for this feature.